Userland/Utilities/test-bindtodevice.cpp

```diff
@@ -107,7 +107,8 @@ void test_send(int fd)
     // bind to an interface that cannot deliver
     char buf[IFNAMSIZ];
     socklen_t buflen = IFNAMSIZ;
-    memcpy(buf, "e1k0", 5);
+    // FIXME: Look up the proper device name instead of hard-coding it
+    memcpy(buf, "ep0s7", 6);
```

Comment:
We could look at the `/proc/net/adapters`. I know this is a test for QEMU, but might still worth it.

Reply:
Well, there's a hard-coded IP address just a few lines after that. I guess we should fix that as well.